packages/browser-repl/config/webpack.config.prod.js

```diff
@@ -0,0 +1,36 @@
+const path = require('path');
+const WebpackVisualizerPlugin = require('webpack-visualizer-plugin');
```

Comment:
did you want this one in `prod` config too, or just `dev` ?

Reply:
only prod

Reply:
But should be excluded from the npm packaging